app/src/main/java/org/freedombox/freedombox/views/fragments/SplashFragment.kt

```diff
@@ -34,12 +39,29 @@ class SplashFragment : BaseFragment() {
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
 
+        openLauncherIfConfigured()
+
         btnSplashNext.setOnClickListener {
             val intent = Intent(activity, DiscoveryActivity::class.java)
             startActivity(intent)
         }
     }
 
+    fun openLauncherIfConfigured() {
+        val configuredBoxesJSON = getSharedPreference(sharedPreferences,
+                getString(R.string.default_box))
+
+        configuredBoxesJSON?.let {
```

Comment:
Try writing a Robolectric test if this activity is launched or not

Reply:
Tests have been included